felix/calc/states_for_test.go

```diff
@@ -1720,19 +1838,46 @@ var vxlanV6NodeResIPDelete = vxlanV6WithBlock.withKVUpdates(
 			Name: remoteHostname,
 		},
 		Spec: apiv3.NodeSpec{BGP: &apiv3.NodeBGPSpec{}}}},
-).withName("VXLAN IPv6 Node Resource IP removed").withVTEPs()
+).withName("VXLAN IPv6 Node Resource IP removed").withRoutes(
+	routeUpdateV6IPPoolVXLAN,
+	proto.RouteUpdate{
+		Type:        proto.RouteType_REMOTE_WORKLOAD,
+		IpPoolType:  proto.IPPoolType_NONE,
+		Dst:         "feed:beef:1::/96",
+		DstNodeName: remoteHostname,
+		//NatOutgoing: true, //TODO: should this be true?
```

Comment:
TODO here should be cleaned up - looks like this should be true? https://github.com/projectcalico/calico/blob/045709221877a7c6d774c2bc1a599c7eec594f81/felix/calc/resources_for_test.go#L703

Reply:
Exactly, so I think there might be a problem, as the route update being produced does not have `NatOutgoing: true`. I'll investigate this and figure what's wrong...

Reply:
Figured it out, it was a problem with the test cases, fixed those. Thanks @caseydavenport!